documentation/docs/developer_guide/git_process.md

```diff
@@ -0,0 +1,13 @@
+# Process to develop and submit new features
+
+This process is the same for any submission to be made to CABLE:
+
+ - a bug fix
+ - a new feature for CABLE with its documentation
+ - a change in the documentation only
+ - etc.
+
+Since the source code and the documentation are held in text files under the same repository, there are no distinction made about updating one or the other. 
```

Comment:
```suggestion
Since the source code and the documentation are held in text files in the same repository, the process for updating them is the same. 
```